java-it-tests/driver21/src/test/java/cassandra/CassandraExecutor21.java

```diff
@@ -27,7 +29,15 @@
     private Session session;
 
     public CassandraExecutor21() {
-        cluster = Cluster.builder().addContactPoint(Config.NATIVE_HOST).withPort(Config.NATIVE_PORT)
+        NettyOptions closeQuickly = new NettyOptions() {
```

Comment:
The java-driver recently changed it's event loop shutdown to allow a (netty default) grace period of 2 seconds before shutting down the event loop to prevent rejected execution exceptions and such.  Adding this cuts ~2 seconds off every test that shuts down a cluster at the end, which was quite a bit.